### PR TITLE
Build bot comments using Jinja2 templates

### PR DESCRIPTION
**Pull request by singingwolfboy**
@sarina

### CODE REVIEW

File: tests/test_pr_comments.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import pytest
+from datetime import datetime
+import openedx_webhooks
+from openedx_webhooks.views.github import (
+    github_community_pr_comment, github_contractor_pr_comment
+)
+
+#pytest.mark.usefixtures('betamax_session')
```
Comment (sarina) [inline tests/test_pr_comments.py]: ?

File: tests/test_pr_comments.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+def make_pull_request(user, number=1, base_repo_name="edx/edx-platform", head_repo_name="edx/edx-platform", created_at=None):
+    "this should really use a framework like factory_boy"
```
Comment (sarina) [inline tests/test_pr_comments.py]: "this is not a real docstring"

**Comment by sarina** [comment]
:+1:

**Comment by singingwolfboy** [comment]
I've discovered that this doesn't work the way I thought it would. The linebreaks in the Jinja2 templates are preserved, and cause the GitHub comments to get formatted weirdly. I might need to write a custom Jinja2 extension to indicate that linebreaks should be stripped -- that sounds like an interesting side project. I'm going to close this PR for now, but keep the branch around.

**Pull request closed without merging**